dev/Expander/Expander_themeresources.xaml

```diff
@@ -338,7 +339,8 @@
                             BorderThickness="{ThemeResource ExpanderChevronBorderThickness}"
                             Background="{ThemeResource ExpanderChevronBackground}">
                             <local:AnimatedIcon
-                                Margin="7"
+                                Width="{StaticResource ExpanderChevronGlyphSize}"
```

Comment:
Just to check, is removing the margin in favor using width and height correct here?

Reply:
I think so, we could have it scale with its parent size by using a margin to limit the size available or set the size explicitly. I think the explicit size makes mores sense here.